go2html.go

```diff
@@ -100,6 +100,12 @@ func (t *Template) Populate(rawReplacements map[string]interface{}) string {
 				result := fragment.template.Populate(nestedReplacement)
 				sb.WriteString(result)
 			}
+		case attrsInjection:
+			rawRepl, _ := rawReplacements[string(fragment)]
+			repl, _ := rawRepl.(map[string]string)
+			for key, val := range repl {
+				sb.WriteString(fmt.Sprintf("%v=%v", key, val))
```

Comment:
attribute value should be within `"..."`

Reply:
also replace `%v` with `%s` formatters

Reply:
I agree, that's a more neat way of building the string. I've fixed that and implemented corresponding requested changes below.